Examples/GenerativeAISample/GenerativeAIMultimodalSample/ViewModels/PhotoReasoningViewModel.swift

```diff
@@ -96,12 +101,18 @@ class PhotoReasoningViewModel: ObservableObject {
 }
 
 private extension CGSize {
-  func fitting(maxSize: CGSize) -> CGSize {
-    if width <= maxSize.width && height <= maxSize.height {
-      return self
+  func fits(inside maxSize: CGSize) -> Bool {
+    return width <= maxSize.width && height <= maxSize.height
+  }
+
+  func aspectFit(inside maxSize: CGSize) -> CGSize {
```

Comment:
Consider documenting that this method assumes a square-ish `maxSize`, or make it square explicitly in the method signature by requiring a single scalar dimension as a parameter. Something like
```
func aspectFit(withLargestDimension dimension: CGFloat) -> CGSize
```

Reply:
That sounds reasonable to me since I expect we'll keep this as a square (and can easily change later if necessary).